src/job/jobHandler.ts

```diff
@@ -218,6 +218,8 @@ export abstract class JobHandler {
     await this._logger.save(this._currJob._id, 'Identified Build dependencies');
     const commands = await downloadBuildDependencies(buildDependencies, this.currJob.payload.repoName, directory);
     await this._logger.save(this._currJob._id, `${commands.join('\n')}`);
+    const err = commands.find((element) => element.includes('ERROR'));
+    if (err) throw new Error('Could not download some dependencies');
```

Comment:
(Not a blocker) Do we want to throw an error if we don't have some build dependencies? I am not sure what the current behavior is. Wonder if some failed build dependencies would still be ok